hip-service-test/Link/Patient/PatientVerificationTest.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using hip_service.Link.Patient;
+using hip_service.OTP;
+using hip_service_test.Link.Builder;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using Moq.Protected;
+using Xunit;
+
+namespace hip_service_test.Link.Patient
+{
+    [Collection("Patient Verification Tests")]
+    public class PatientVerificationTest
+    {
+        private readonly IConfiguration configuration;
+        private PatientVerification patientVerification;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'patientVerification' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4807294)